### DISCUSSION TITLE
Even I followed through the first tutorial I didn't get to make it work successfully

### DISCUSSION THREAD

**Discussion by thsnyzkn** [tailwindlabs/tailwindcss#7820] [Help]
Hey everyone,
I was trying the tutorial from the installation page(https://tailwindcss.com/docs/installation) using Tailwind CLI.
I followed the steps, but in order to see the respective styles that are added by Tailwind I needed to change the link's href in the head from ` <link href="/dist/output.css" rel="stylesheet">` to `<link href="../dist/output.css" rel="stylesheet" />`.
Did I make anything wrong or the content of the tutorial needs to be changed?

Thanks!

**Comment by skimhugo** [comment]
I think if you are a starter with tailwind, you should find a starter project to begin your journey.

**Comment by AviAvinav** [comment]
I followed the same tutorial as the one you mentioned. It seems to work in my case, I didn't have to change it to `../dist`. The problem might be with your project structure.

My project structure was as follows:

dist
node_modules
src
package.json
tailwind.config.js
yarn.lock

dist folder includes the output.css file and src contains index.html and input.css.

**Reply by thsnyzkn** [reply]
Interesting, I didn't use npm or yarn to init a project I just used mentioned commands within the tutorial and my folder structure is following:

dist(output.css)
src(index.html, input.css)
tailwind.config.js

**Reply by AviAvinav** [reply]
Even if you didn't use yarn or npm init, it should work the same because tailwindcss is a dev dependency. I tried it without initializing with npm or yarn with the same folder structure as you. It still seems to work in my case.

Though one thing I did notice was that once I added a new class and it rebuild the output.css, it didn't make changes on the site. In that case, you can try reloading the page, and if still doesn't work try restarting the dev server, because if `../dist` is working for you,` /dist` should also work as they are essentially the same thing.
